modules/core/client/components/ReadMorePanel.js

```diff
@@ -0,0 +1,50 @@
+// External dependencies
+import { withTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+
+// Internal dependencies
+import '@/config/client/i18n';
+import { plainText, plainTextLength } from '../utils/filters';
+
+const LIMIT = 2000;
+
+function ReadMorePanel({ content, id, t }) {
+  const [showMore, setShowMore] = useState(false);
+
+  if (content.length === 0) {
+    return null;
+  }
+
+  if (showMore || plainTextLength(content) <= LIMIT) {
+    return <div dangerouslySetInnerHTML={{ __html: content }} />;
```

Comment:
Just noting that things are sanitized at the backend before outputting via APIs.

Reply:
Related convo: https://github.com/Trustroots/trustroots/pull/1161#discussion_r363119558